dom0/sd-whonix-rsyslog-enable.sls

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+# vim: set syntax=yaml ts=2 sw=2 sts=2 et :
+
+##
+# Enables securedrop-log rsyslog plugin in sd-whonix
+##
+
+sd-rsyslog-for-sd-whonix:
+  file.managed:
+    - name: /rw/config/sd-rsyslog.conf
+    - source: "salt://sd-rsyslog.conf.j2"
+    - template: jinja
+    - context:
+        vmname: sd-whonix
+
+sd-rsyslog-sdlog-conf-for-sd-whonix:
```

Comment:
Can we add a comment here, for future maintainers, as to why this is required here and not in other VMs, where this file is handled by the sd-log package: https://github.com/freedomofpress/securedrop-log/blob/d1799b2477f4250eae43b472472df19a49f5cb75/sdlog.conf